src/library-authoring/import-course/messages.ts

```diff
@@ -164,6 +164,61 @@ const messages = defineMessages({
+    description: 'Title of the info card when course import analysis is complete and all data can be imported.',
+  },
+  importCourseAnalysisCompleteAllContentBody: {
+    id: 'library-authoring.import-course.review-details.analysis-complete.100.body',
+    defaultMessage: 'All course content will imported into a collection in your library called {courseName}. See details below.',
+    description: 'Body of the info card when course import analysis is complete and all data can be imported.',
+  },
+  importCourseAnalysisCompleteSomeContentTitle: {
+    id: 'library-authoring.import-course.review-details.in-progress.title',
+    defaultMessage: 'Import Analysis Complete',
+    description: 'Title of the info card when course import analysis is complete and some data can be imported.',
+  },
+  importCourseAnalysisCompleteSomeContentBody: {
+    id: 'library-authoring.import-course.review-details.analysis-complete.100.body',
+    defaultMessage: '{unsupportedBlockPercentage}% of content cannot be imported. For details see below.',
+    description: 'Body of the info card when course import analysis is complete and some data can be imported.',
+  },
+  importCourseAnalysisDetailsUnsupportedBlocksBody: {
+    id: 'library-authoring.import-course.review-details.analysis-details.unsupportedBlocks.body',
+    defaultMessage: 'The following block types cannot be imported into your library because they\'re are not yet supported. These block types will be replaced with a placeholder block in the library. For more information, reference the Help & Support sidebar.',
+    description: 'Body of analysis details when some unsupported blocks are present',
+  },
+  importCourseComponentsUnsupportedInfo: {
+    id: 'library-authoring.import-course.review-details.analysis-summary.components.unsupportedInfo',
+    defaultMessage: '{count, plural, one {{count} block} other {{count} blocks}} unsupported',
+    description: 'Message to display on hovering over info icon near components number for unsupported blocks',
+  },
+  importCourseAnalysisDetails: {
+    id: 'library-authoring.import-course.review-details.analysis-details.title',
+    defaultMessage: 'Analysis Details',
+    description: 'Title of the card for the analysis details of a imported course.',
+  },
+  importCourseAnalysisCompleteReimportTitle: {
+    id: 'library-authoring.import-course.review-details.analysis-complete.reimport.title',
+    defaultMessage: 'Import Analysis Completed: Reimport',
+    description: 'Title of the info card when course import analysis is complete and it was already imported before.',
+  },
+  importCourseAnalysisCompleteReimportBody: {
+    id: 'library-authoring.import-course.review-details.analysis-complete.reimport.body',
+    defaultMessage: '{courseName} has already been imported into the Library "{libraryName}". If this secion is re-imported, all Sections, Subsections, Units and Content Blocks will be reimported again.',
```

Comment:
@edschema CC @navinkarkera I think it should be "course" instead of "section"

```suggestion
    defaultMessage: '{courseName} has already been imported into the Library "{libraryName}". If this course is re-imported, all Sections, Subsections, Units and Content Blocks will be reimported again.',
```

<img width="1444" height="208" alt="image" src="https://github.com/user-attachments/assets/dc37139e-58e1-4d46-ad06-ec361ed9a86a" />